### DISCUSSION TITLE
thanhle.blog/bai-moi-chuyen-lam-product

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#87] [Ideas]
# thanhle.blog/bai-moi-chuyen-lam-product

https://thanhle.blog/blog/bai-moi-chuyen-lam-product

**Comment by hai123vn** [comment]
lại là em đây, những bài blog của anh rất bổ ích và hay lắm ạ. Chúc anh thật nhiều sức khỏe để hoàn thành những mục tiêu tiếp theo nha anh. Đặc biệt ra bài đều đều anh nhé

**Comment by thanhlmm** [comment]
Thực ra mình có nhiều topic/idea để nói lắm, tất cả mình đều đề ở đây https://thanh-le.notion.site/c0a9456d6fa04bb2af554a310ac7b5ff?v=34aad7277ebc4924ae16a897baad7849

Mọi người thấy phần nào muốn nghe góc nhìn của mình thì comment nhé. Mình ko giám chắc là sẽ có câu tl hoặc sẽ là một bài blog vì nhiều cái chỉ là idea, nhiều cái thì nghĩ là mình biết nhưng tới khi vào làm vài gạch đâu dòng thì té ra mình "ảo tưởng". But anyway, give me your thought
